CHANGELOG.md

```diff
@@ -16,7 +21,21 @@
 - `ChatThreadClient` `remove(participant)` method now accepts `CommunicationIdentifier` instead of a string
 - For `Participant` renamed `user` property to `id`
 
+**Azure Communication Calling**
+- `Renderer` renamed to `VideoStreamRenderer`.
```

Comment:
Should this be considered a breaking change?

Reply:
Yes, as it requires code change from the end user perspective to keep using the SDK?

Reply:
Ah sorry I misinterpreted looked like it was in the wrong section my mistake!